src/content/release/breaking-changes/navigator-and-page-api.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Navigator's page APIs breaking change
+description: >
+  Changes to Navigator's page APIs.
```

Comment:
Could you add a word or two to be more specific here, as the APIs might have other breaking changes in the future.

Reply:
good point